repose-aggregator/services/service-client/impl/src/main/java/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImpl.java

```diff
@@ -60,29 +61,39 @@ public UntypedActor create() {
 
 
     @Override
-    public ServiceClientResponse get(String hashKey, String uri, Map<String, String> headers) {
+    public ServiceClientResponse get(String hashKey, String uri, Map<String, String> headers) throws TimeoutException {
```

Comment:
this doesn't solve the npe, it only handles it in the case of a timeout exception. if there is something else exceptional you are still going to potentially return null. This where we do what i was talking about during sprint planning (maybe it didn't get captured?). Instead of only dealing with the timeout exception, we catch all exceptions, and wrap them in an akka service client specific exception. Should the caller care to take different action depending on what the underlying exception was they can dig it out.